CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Fix typo in conflicting lockfile failure message ([#901](https://github.com/heroku/heroku-buildpack-nodejs/pull/901))
 - Use local packages with Yarn when present ([#913](https://github.com/heroku/heroku-buildpack-nodejs/pull/913))
 - Add support for running "npm ci" when "npm-shrinkwrap.json" is found on the project directory ([#938](https://github.com/heroku/heroku-buildpack-nodejs/pull/938))
+- Support non-zero-install support for Yarn 2 ([#888](https://github.com/heroku/heroku-buildpack-nodejs/pull/888))
```

Comment:
@joshwlewis This changelog entry is under the already-released v187 rather than `main`

Reply:
Correcting this in https://github.com/heroku/heroku-buildpack-nodejs/pull/941/commits/21a6d7e856e16f080ff5990de0f75b722ac36a34.